examples/with-fauna/actions/entry.js

```diff
@@ -0,0 +1,21 @@
+'use server'
```

Comment:
```suggestion
'use server'

```

Reply:
I can't commit changes back to kick off a new build, so just make sure prettier/linting pass locally first 🙏

Reply:
Updated as per suggestion and ran prettier.